projects/allen-cell-segmenter.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Allen Cell Segmenter
+layout: page
+---
+
+- *Contributor(s)*: Jianxu Chen (@jxchen01), Benjamin Morris (@benjijamorris)
+- *References*: [Project Site](https://www.allencell.org/segmenter.html), [Github](https://github.com/AllenCell/aics-segmentation), [Publication](https://www.biorxiv.org/content/10.1101/491035v2), [Support](https://forum.allencell.org/c/software-code/segmenter/16)
+
+## Purpose
+
+> Describe the problem you are trying to solve by packaging your tool as a napari plugin in 1-2 sentences.
+I should be able to read this alone and communicate the value to someone else.
+
+### Why does this matter to biologists?
+
+### Why does this matter to you and your professional goals and responsibilities?
+
```

Comment:
The development of this plugin will enable a broader community of scientists to perform 3D segmentation on their images and aligns with the mission of the [Allen Institute for Cell Science](allencell.org) for open science.